app/src/main/java/nerd/tuxmobil/fahrplan/congress/alarms/AlarmTimePickerFragment.kt

```diff
@@ -0,0 +1,109 @@
+package nerd.tuxmobil.fahrplan.congress.alarms
+
+import nerd.tuxmobil.fahrplan.congress.R
+import nerd.tuxmobil.fahrplan.congress.extensions.getLayoutInflater
+import nerd.tuxmobil.fahrplan.congress.extensions.requireViewByIdCompat
+import nerd.tuxmobil.fahrplan.congress.BuildConfig
+import nerd.tuxmobil.fahrplan.congress.repositories.AppRepository
+
+import android.os.Bundle
+import android.app.Dialog
+import android.view.View
+import android.widget.Spinner
+import android.widget.ArrayAdapter
+import android.content.DialogInterface
+import android.annotation.SuppressLint
+
+import androidx.annotation.MainThread
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentResultListener
+
+import java.lang.NullPointerException
+
+class AlarmTimePickerFragment : DialogFragment() {
+    private lateinit var spinner: Spinner
+    private var alarmTimeIndex = 0
```

Comment:
An empty line before and after the fields would please my eyes.
Less empty lines within the functions below would be less clutter in my opinion.